package.json

```diff
@@ -28,6 +33,9 @@
     "tailwind:status": "node ./scripts/tailwind-status.js",
     "codegen": "graphql-codegen --config codegen.yml"
   },
+  "resolutions": {
+    "@types/react": "^17.0.0"
```

Comment:
Is there a reason why you are using this? the frontend is expecting react 18

Reply:
That is old code, from the first time Jonas worked on it. I see now, there is no `resolutions` in the most recent `package.json`. Thanks for spotting it